leapp/utils/report.py

```diff
@@ -18,3 +18,18 @@ def fetch_upgrade_report_messages(context_id):
         messages.append(report)
 
     return messages
+
+
+def generate_report_file(messages_to_report, path):
+    if path.endswith(".txt"):
+        with open(path, 'w') as f:
+            for message in messages_to_report:
+                f.write('Severity: {}\n'.format(message['severity']))
+                f.write('Title: {}\n'.format(message['title']))
+                f.write('Summary: {}\n'.format(message['summary']))
+                if message.get('detail', {}).get('remediation'):
+                    f.write('Remediation: {}\n'.format(message['detail']['remediation']))
```

Comment:
Ditto from above, and apart from that `remediations` is a list of possible remediations in `{type: ..., value: ...}` format so we probably want to do something like this here
```
['{type}\n  {value}'.format(**r) for r in message['detail']['remediations']]
```